official/docs/curl/billing/create-easypost-credit-card.sh

```diff
@@ -0,0 +1,9 @@
+curl -X POST "https://api.easypost.com/v2/credit_cards" \
+  -u "<REFERRAL_USER_API_KEY>:" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "credit_card": {
+      "stripe_object_id":"tok_...",
+      "priority":"primary"
```

Comment:
JSON needs to have their keys/values separated by a space. Linter doesn't catch this because free-form JSON in a curl command can't be linted easily by our linter.

```suggestion
      "stripe_object_id": "tok_...",
      "priority": "primary"
```

Reply:
resolved.